src/map/pc.cpp

```diff
@@ -6454,6 +6468,11 @@ enum e_setpos pc_setpos(struct map_session_data* sd, unsigned short mapindex, in
 	}
 	else 
 		sd->count_rewarp = 0;
+
+	if (sd->state.vending)
+		vending_update_vendor_location(*sd);
+	else if (sd->state.buyingstore)
```

Comment:
```suggestion
	if (sd->state.buyingstore)
```
Just for consistency with old code.